mmrazor/datasets/utils.py

```diff
@@ -1,15 +1,63 @@
+import os
+import random
+
 from torch.utils.data import random_split
 
 
-def split_dataset(dataset):
+def split_dataset_txt(txt_path, save_dir):
+    index_pool = dict()
+    with open(txt_path, 'r') as f:
+        for line in f:
+            line = line.strip()
+            (filename, index) = line.split(' ')
+            if index not in index_pool:
+                index_pool[index] = []
+            index_pool[index].append(filename)
+
+    assert len(index_pool) == 1000
+
+    train_txt = list()
+    val_txt = list()
```

Comment:
The `train/val_txt` should be `part_xx`.
This script can also be reused when supporting semi-supervised algorithms in the future.